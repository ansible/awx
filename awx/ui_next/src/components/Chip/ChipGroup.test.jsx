import React from 'react';
import { mount } from 'enzyme';
import { ChipGroup } from '.';

describe('<ChipGroup />', () => {
  test('renders the expected content', () => {
    const wrapper = mount(<ChipGroup />);
    expect(wrapper).toMatchSnapshot();
  });
});
