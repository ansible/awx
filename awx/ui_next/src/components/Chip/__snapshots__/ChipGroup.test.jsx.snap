// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ChipGroup /> renders the expected content 1`] = `
<ChipGroup>
  <StyledComponent
    forwardedComponent={
      Object {
        "$$typeof": Symbol(react.forward_ref),
        "attrs": Array [],
        "componentStyle": ComponentStyle {
          "componentId": "ChipGroup-sc-10zu8t0-0",
          "isStatic": true,
          "lastClassName": "bwbBYO",
          "rules": Array [
            "--pf-c-chip-group--c-chip--MarginRight:10px;--pf-c-chip-group--c-chip--MarginBottom:10px;> .pf-c-chip.pf-m-overflow .pf-c-button{--pf-c-button--PaddingTop:3px;--pf-c-button--PaddingBottom:3px;--pf-c-chip--m-overflow--c-button--PaddingLeft:8px;--pf-c-chip--m-overflow--c-button--PaddingRight:8px;}",
          ],
        },
        "displayName": "ChipGroup",
        "foldedComponentIds": Array [],
        "render": [Function],
        "styledComponentId": "ChipGroup-sc-10zu8t0-0",
        "target": [Function],
        "toString": [Function],
        "warnTooManyClasses": [Function],
        "withComponent": [Function],
      }
    }
    forwardedRef={null}
  >
    <ChipGroup
      className="ChipGroup-sc-10zu8t0-0 bwbBYO"
      collapsedText="\${remaining} more"
      defaultIsOpen={false}
      expandedText="Show Less"
      numChips={3}
      withToolbar={false}
    />
  </StyledComponent>
</ChipGroup>
`;
